frontend/src/components/category-import/CategoryImport.vue

```diff
@@ -434,17 +447,35 @@ function importParticipatingJudges() {
   }
 }
 
-function importShortTexts(key: string) {
-  if (isDecision(targetDocumentUnit.value)) {
-    const source =
-      sourceDocumentUnit.value?.shortTexts[
-        key as keyof typeof sourceDocumentUnit.value.shortTexts
-      ]
+function importDecisionNames() {
+  if (
+    isDecision(targetDocumentUnit.value) &&
+    isDecision(sourceDocumentUnit.value)
+  ) {
+    const source = sourceDocumentUnit.value?.shortTexts["decisionNames"]
+    if (!source) return
+    //source.forEach((judge) => (judge.id = undefined))
+
+    targetDocumentUnit.value!.shortTexts["decisionNames"] = [...source]
+  }
+}
+
+// By narrowing the type of key to exclude "decisionNames", TypeScript no longer considers the possibility of assigning a non-string value to documentUnit.value.shortTexts[key].
+type ShortTextKeys =
```

Comment:
```suggestion
type ShortTextStringKeys =  Omit<keyof typeof shortTextLabels, "decisionName">
```
Vielleicht könnte man hier noch einen Namen finden, der ausdrückt, dass es nicht nur `keyOf ShortText` ist. Mir ist aber nix besseres eingefallen.

Statt alles aufzulisten, könnte man hier auch mit `Omit` arbeiten.